ikura/chain/node/src/chain_spec/res/gondatsu.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "Kusama Blobs",
-  "id": "blobs_kusama",
+  "name": "Gondatsu",
+  "id": "gondatsu",
   "chainType": "Live",
   "bootNodes": [
     "/dns/blob-kusama-collator-archive-1.thrum.dev/tcp/30333/p2p/12D3KooWAn4jHXxvSKLtDn5QSCyE1CMbpcLfdUksrzLF8NrMSA4W",
     "/dns/blob-kusama-rpc-bootnode-1.thrum.dev/tcp/30333/ws/p2p/12D3KooWFjktFNpwTc8TMo54H9sGhTTHyp2WdaZ11YJ6mt2NjfkZ"
   ],
   "telemetryEndpoints": null,
-  "protocolId": "sugondat-kusama",
+  "protocolId": "gondatsu",
```

Comment:
https://paritytech.github.io/polkadot-sdk/master/sc_network/config/struct.ProtocolId.html

This is deprecated and has no effect actually. I tested locally and my node synced without any problem.